### ISSUE TITLE
Activation flow for new users

### ISSUE THREAD

**Issue by buzzard10** [goauthentik/authentik#5799]
**Describe your question/**
I've created flow which works for activation of new users. Through API i'm using endpoint `/core/users/{id}/recovery/` to generate new link for password set. I would like to know how to use `GET /core/users/{id}/recovery_email/`. I've added to my flow email stage but it doesn't work. Any ideas? Here you have diagram:

**Screenshots**
![image](https://github.com/goauthentik/authentik/assets/17282191/5575f68f-b9e4-492b-829a-fffda3d4849e)

**Comment by pinarruiz** [comment]
I am guessing you are sending this email to the user itself right?

**Comment by buzzard10** [comment]
@pinarruiz i would like to, but while i'm posting an API request, i have error "Server error"

**Comment by pinarruiz** [comment]
If i am not wrong, maybe try to disable the superuser requirement (Edit -> Authentication = No Requirements). I dont still know if I am getting the use case for the flow right.

**Comment by buzzard10** [comment]
@pinarruiz it doesn't work ;c

**Comment by pinarruiz** [comment]
Could you explain a little bit more how the flow is meant to be used?

**Comment by buzzard10** [comment]
Of course. I'm creating an integration with CRM which will automatically create user account in authentik after i'll add new customer to crm. I want to automatically generate account and send link to a customer which will allow him to set password for his account. I can do it now with temporary link (first endpoint mentioned above), but then i have to send this link to a customer by myself. So i'm trying to do this through authentik

**Comment by BeryJu** [comment]
@buzzard10 The "Server error" response shouldn't happen, are there any error messages in the server container's logs? Looking at the diagram of the flow you posted, it looks correct. The endpoint you mentioned above also requires a POST request now.
